### PR TITLE
Update driver-kafka to 2.8.1 and prepare for Kafka JMS testing

### PR DESCRIPTION
**Pull request by dave2wave**
Update Kafka to 2.8.1
Update to openjdk11
Use AWS Spot Instances
Configure driver-jms/kafka*yaml drivers
Change download location away from an ASF Mirror
No autocommit in Kafka Consumer

### CODE REVIEW

File: driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java [hunk 1]
```diff
@@ -47,11 +45,11 @@
     private final ExecutorService executor;
     private final Future<?> consumerTask;
     private volatile boolean closing = false;
-    private boolean autoCommit;
-    public KafkaBenchmarkConsumer(KafkaConsumer<String, byte[]> consumer, Properties consumerConfig, ConsumerCallback callback) {
+
+    public KafkaBenchmarkConsumer(KafkaConsumer<String, byte[]> consumer, ConsumerCallback callback) {
```
Comment (merlimat) [inline driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java:49]: This change is reverting #148. Is there any reason to do so?

Comment (dave2wave) [reply driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java:49]: I'll need to discuss with @eolivelli

File: driver-kafka/deploy/ssd-deployment/provision-kafka-aws.tf [hunk 2]
```diff
@@ -120,32 +120,34 @@ resource "aws_instance" "zookeeper" {
...
   count                  = "${var.num_instances["kafka"]}"
+  wait_for_fulfillment   = true
 
   tags = {
     Name = "kafka-${count.index}"
   }
 }
 
-resource "aws_instance" "client" {
+resource "aws_spot_instance_request" "client" {
```
Comment (eolivelli) [inline driver-kafka/deploy/ssd-deployment/provision-kafka-aws.tf:137]: please revert this part about using "spot instances"
IIUC this is not required for the Kafka upgrade

File: driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java (hunk 1 above)
Comment (eolivelli) [reply driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java:49]: I don't see any reason to revert this.

My understanding is that simply @dave2wave in our fork we did not have #148 
I would simply revert this change here in this PR

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java (hunk 1 above)
Comment (dave2wave) [reply driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java]: Done

**Review by merlimat — APPROVED** [review]

**Pull request merged**